config/locales/es.yml

```diff
@@ -1448,110 +1449,110 @@ es:
+          all_copies_html: 'Tengo  <u>todas</u> las copias de mis documentos fiscales oficiales '
+          does_not_apply_html: "<u> No tengo </u> copias de ningún documento fiscal oficial porque <u> no me aplican </u>"
+          need_help_html: "<u> No tengo</u> ninguna copia de mis documentos fiscales oficiales y <u> me gustaría recibir ayuda </u> para obtenerlas."
+          some_copies_html: 'Tengo <u>algunas</u> copias de mis documentos fiscales oficiales '
+        help_text: Los documentos fiscales se refieren a cualquier documento oficial del IRS o documentos que confirmen su empleo como una W2, 1099, registro de trabajo personal, etc.
         title: "¿Qué tipo de documentos fiscales tiene?"
     triage_express:
       edit:
-        collect_your: 'Recoge tu:'
+        collect_your: 'Cobre su:'
         collectables:
-          additional_credits: Créditos Estatales Adicionales
+          additional_credits: Créditos estatales adicionales
           ctc: Crédito Tributario por Hijos
           eitc: Crédito tributario por ingresos del trabajo
-          missing_stimulus: Faltan Pagos de Estímulo
+          missing_stimulus: Pagos de estímulos faltantes
         comparison:
           express:
-            description: "¡Archive rápidamente por su cuenta!"
-            timeframe: 15 minutos para presentar, disponible en marzo de 2022
-            title: Express
+            description: "¡Declare rápido por cuenta propia!"
+            timeframe: 15 minutos para declarar sus impuestos, disponible abril 2022
+            title: Express/Rápido
           gyr:
-            description: "¡Recoge el reembolso más grande! Presente con la ayuda de nuestros voluntarios certificados por el IRS."
-            timeframe: 2-3 semanas para presentar
+            description: "¡Cobre el reembolso más grande! Presente su declaración con asistencia de nuestros voluntarios certificados por IRS."
+            timeframe: De 2-3 semanas para declarar sus impuestos
             title: GetYourRefund
-        file_with_gyr: Archivo con GetYourRefund
+        file_with_gyr: Solicite por medio de GetYourRefund
         refund: Reembolso
-        sign_up_for_express: Suscríbete a Express
-        title: "¡Tenemos dos opciones gratuitas que pueden funcionar para usted!"
+        sign_up_for_express: Subscríbase para Express
+        title: "¡Tenemos dos opciones gratis que pueden funcionar para usted!"
     triage_gyr:
       edit:
-        file_in_person: Presentar en persona
-        file_on_my_own: En realidad, prefiero presentar la solicitud rápidamente por mi cuenta sin ayuda.
-        file_online: Archivo en línea
+        file_in_person: Presente su declaración en persona
+        file_on_my_own: 'En realidad, quiero intentar presentar mi declaración rápidamente por mi cuenta utilizando software gratuito. '
```

Comment:
There is an unnecessary ` ` (space) character at the end of this string. It is not a problem, but it is unnecessary.